index.html

```diff
@@ -0,0 +1,123 @@
+          Supporting Contents
+        </div>
+        <div class="col-6 col-md-12">
+          To aid your studies, we have compiled a list of tutorials which we deem best from the web.
+        </div>
+      </div>
+       <div class="row flex-md-column row3">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/terminal.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Programming Tasks
+        </div>
+        <div class="col-6 col-md-12">
+          Coding tasks, mainly javascript function coding, along with the associative hints and answers.
+        </div>
+      </div>
+    </div>
+    <button type="button" class="sub m-3 mb-5 btn getstarted">Get started with the coding Challenges</button>
+  </section>
+  <section class="about" id="about">
+    <div class="mainImg">
+      <h3 class="motto d-flex justify-content-center text-center pt-5">"Preparing you for the language of the future!"</h3>
+      <h1 class="description-header d-flex justify-content-center text-center pt-3">CODING CHALLENGES</h1>
+      <div class="about-description text-center">
+        <p class="pt-2">Coding challenges is designed to test your knowledge on programming languages and give context on how well you are performing. In the future, leaderboard will be integrated to track your performance compared to other programmers.</p>
+      </div>
+      <div class="about-passage lato text-center">
+        Please contact us for any further questions about CC <br>
+        <a class="text-decoration-none fw-bold text-dark" href="#">debmirmij@gmail.com</a>
+      </div>
+    </div>
+    <div class="d-flex flex-column justify-content-center">
+      <h1 class="coding-header mt-5">Coding Challenges Logo</h1>
+      <div class="d-flex justify-content-center">
+        <img src="images/cc-logo-large.png" width="400" alt="large logo">
+      </div>
+    </div>
+    <div class="partner mt-5 pb-4 d-flex flex-column justify-content-evenly">
+      <h1 class="coding-header partner-header mt-5">Partner</h1>
+      <div class="d-flex align-items-center justify-content-evenly"><a href="https://www.microverse.org/"><img src="images/microverse-transparent.png" width="150" alt="microverse"></a><a href="https://github.com/"><img src="images/github-transparent.png" width="130" alt="github"></a></div>
+    </div>
+  </section>
+  <footer class="foot d-flex justify-content-center mt-4 p-4 mx-auto m-auto">
+    <img src="images/cc-logo-ethiopia.png" width="100" alt="logo-footer">
+    <span>&copy; 2022. Some rights reserved.</span>
+  </footer>
+  </div>
+  <script src="scripts/data.js"></script>
+  <script src="scripts/menu.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa" crossorigin="anonymous"></script>
```

Comment:
- [ ] [REQUIRED] You did a great job in the coding challenges section, but this is a feature. I still don't see the `speakers section`, please create it like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

#

![image](https://user-images.githubusercontent.com/63915024/181352481-f71d6d05-eea9-4dca-9188-42b262da155e.png)